app/models/foreman_docker/docker.rb

```diff
@@ -131,17 +132,19 @@ def bootstrap(args)
     end
 
     def client
-      @client ||= ::Fog::Compute.new(
+      cli = ::Fog::Compute.new(
```

Comment:
Really this is a limitation of the docker-api. I've opened an issue but it would require reworking them gem:

https://github.com/swipely/docker-api/issues/253

I'm tempted to fork it.